Dockerfile

```diff
@@ -12,23 +12,26 @@ RUN apt-get update && \
 
 # add group & user
 ARG USER_HOME=/home/piper
+ARG CFCLI_VERSION
```

Comment:
you can still overwrite the version in case you don't want to use the one from github via:
`docker build --build-arg CFCLI_VERSION=8.4.0 ...`